pages/dkp/kommander/1.3/projects/project-network-policies/index.md

```diff
@@ -0,0 +1,116 @@
+
+Suppose you need to create a network policy to allow incoming traffic to API service pods in a specific Kommander Project's namespace from any other pod in any namespace that has the label, service.corp/users-api-role: client. For this example, API service pods are those pods that were created with the Label, service.corp/users/role: API.
+
+You can limit the policy to just incoming traffic from select namespaces by adding an ingress rule with these characteristics:
+
+- Use Port 8080 to receive incoming TCP traffic, and...
+- Refuse traffic from pods unless they are client pods that have a Label, such as `service.corp/users/role: client`, which follows a common microservice architecture pattern, "microservice-tier-role: accessmode".
+
+### Provide the General Information
+1. Select **Projects** from the left-hand navigation bar, then select the **Network Policy** tab.
+2. Type "microsvc-users-API-allow" in the **ID Name** field.
+3. Type “Allow Users microservice clients to reach the APIs provided in this namespace” in the **Description** field.
+4. Select **Add** under **Pod Selector** and then select **Match Labels**.
+5. Set the **Key** to service.users/role and the **Value** to API.
+
+### Create an Ingress rule
+1. Leave **Policy Types** set to Default.
+2. Scroll down to **Ingress Rules** and select **+ Add an Ingress Rule**.
+3. Select **+ Add Port**, and set the **Port** to 8080 and the **Protocol** to TCP.
+
+### Add Sources
+1. Select **+ Add Sources** and mark the **Select All Namespaces** check box.
+2. Select **+ Add Pod Selector**.
+3. Select **Match Labels**.
+4. Set the **Key** value to users select all names bases service.corp/users/role and set the **Value** to client.
+5. Scroll up and select **Save**.
+
+## Ingress: Limit pods that access a database to this namespace
+Suppose that while deploying an application in a project, you want to protect its database pods by allowing ingress from API service pods in the current namespace, and prevent ingress from pods in any other namespace.
+
+You can limit the database pods to just incoming traffic from the current namespaces by adding an ingress rule with these characteristics:
+
+### Provide the General Information
+1. Select **Project** from the left-hand navigation bar, then select the **Network Policy** tab.
+2. Type database-access-api-only in the **ID name** field.
+3. Type “Allow MySQL access only from API pods in this namespace" in the **Description** field.
+4. Select **Pod Selector** then select **Match Labels**.
+5. Set the Key to tier and the Value to database.
+
+### Create an Ingress rule
+1. Scroll down and select Default for the **Policy Types**.
+2. Scroll down to the Ingress section.
+3. Select **+ Add Ingress Rule**, then select **+ Add Port**.
+4. Set the **Port** to 3306 and leave the **Protocol** set to TCP.
+
+### Add Sources
+1. Select **+ Add Source**.
+2. Select **+ Add Pod Selector** and set the **Key** to tier and the **Value** to API.
```

Comment:
Maybe add something about selecting Match Labels between adding pod selector and entering the values?